pkg/runner/runner.go

```diff
@@ -64,7 +69,7 @@ func NewRunner(ctx context.Context, logger *log.Logger, insecure bool, goCmd str
 
 	goVersion, err := parseGoVersion(output.String())
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "failed to parse go version")
```

Comment:
```suggestion
		return nil, errors.Wrap(err, "parse go version")
```
We don't need to mentioned it failed. We are wrapping error after all. (:

Reply:
Fixed!